### ISSUE TITLE
Avatar upload doesn't work

### ISSUE THREAD

**Issue by drnasin** [david-szabo97/grav-plugin-admin-addon-user-manager#48]
I'm trying to add a new avatar for a user but it doesn't work. After clicking save image is not saved in page folder and is basically non existant. Tried png and jpg. Same result. No error message reported..

**Comment by david-szabo97** [comment]
@drnasin Hah, I am clueless. Tried to find the root of the issue but failed...

**Comment by drnasin** [comment]
@david-szabo97 I see this is still not fixed. 
Well I'm not sure what's the logic behind your plugin but If I add this code (manually) to the generated user file it works…maybe that will help...

```
avatar:
  user/accounts/avatars/7xTltLVA3dw0JN8.png:
    name: 7xTltLVA3dw0JN8.png
    type: image/png
    size: 31977
    path: user/accounts/avatars/7xTltLVA3dw0JN8.png
```

**Comment by david-szabo97** [comment]
Fixed in v2.2.0
